fastcrud/endpoint/endpoint_creator.py

```diff
@@ -310,17 +320,39 @@ def _read_items(self):
 
         async def endpoint(
             db: AsyncSession = Depends(self.session),
-            offset: int = Query(0),
-            limit: int = Query(100),
+            page: Optional[int] = Query(
+                None, alias="page", description="Page number"
+            ),
+            items_per_page: Optional[int] = Query(
+                None, alias="itemsPerPage", description="Number of items per page"
+            ),
             filters: dict = Depends(dynamic_filters),
         ):
-            return await self.crud.get_multi(db, offset=offset, limit=limit, **filters)
+            if not (page and items_per_page):
+                return await self.crud.get_multi(db, offset=0, limit=100,
+                                                 **filters)
+
+            offset = compute_offset(page=page, items_per_page=items_per_page)
+            crud_data = await self.crud.get_multi(
+                db, offset=offset, limit=items_per_page, **filters
+            )
+
+            return paginated_response(
+                crud_data=crud_data, page=page, items_per_page=items_per_page
+            )  # pragma: no cover
 
         return endpoint
 
     def _read_paginated(self):
         """Creates an endpoint for reading multiple items from the database with pagination."""
         dynamic_filters = _create_dynamic_filters(self.filter_config, self.column_types)
+        warnings.warn(
+            "_read_paginated endpoint is getting deprecated and mixed "
+            "into _read_items. You can keep using _read_items without changes "
```

Comment:
You should just add here that it will be dropped in the next "major" update (so people know exactly when it will stop working before upgrading)

Reply:
Do we want to mention some specific version?